this was commented out for some reason, making it ignore the CFLAGS/CXXFLAGS we pass

--- a/src/3rdparty/gn/build/gen.py
+++ b/src/3rdparty/gn/build/gen.py
@@ -428,15 +428,10 @@
   #  ar = os.environ.get('AR', 'ar')
 
 
-  # cflags = os.environ.get('CFLAGS', '').split()
-  # cflags += os.environ.get('CXXFLAGS', '').split()
-  # ldflags = os.environ.get('LDFLAGS', '').split()
-  # libflags = os.environ.get('LIBFLAGS', '').split()
-
-  cflags = []
-  cflags_cc = []
-  ldflags = []
-  libflags = []
+  cflags = os.environ.get('CFLAGS', '').split()
+  cflags += os.environ.get('CXXFLAGS', '').split()
+  ldflags = os.environ.get('LDFLAGS', '').split()
+  libflags = os.environ.get('LIBFLAGS', '').split()
 
   cc = options.cc
   cxx = options.cxx
